armi/cases/case.py

```diff
@@ -142,7 +142,7 @@ def bp(self):
         This property allows lazy loading.
         """
         if self._bp is None:
-            self._bp = blueprints.loadFromCs(self.cs)
+            self._bp = blueprints.loadFromCs(self.cs, roundTrip=True)
```

Comment:
why true by default here but false by default in the method? Could use a comment explaining

Reply:
The idea here is that non-round-trip is the much more common case in our code base.  Normally, we are just reading data from blueprint YAMLs.  But in the caseSuite situation, we might want to write them back out, so in that one scenario we want 'round trip' as an option.

Am I wrong in my understanding of the situation?